geolocator_platform_interface/lib/src/implementations/method_channel_geolocator.dart

```diff
@@ -25,13 +25,19 @@ class MethodChannelGeolocator extends GeolocatorPlatform {
   EventChannel eventChannel =
       EventChannel('flutter.baseflow.com/geolocator_updates');
 
+  /// The event channel used to receive [NmeaMessage] updates from the native
+  /// platform.
+  @visibleForTesting
+  EventChannel nmeaChannel = EventChannel('flutter.baseflow.com/nmea_updates');
```

Comment:
Similar to the implementation on the JAVA side, I think here we should also use clear naming to differentiate between the event channel broadcasting positions and the event channel broadcasting NMEA messages:

```suggestion
  EventChannel positionEventChannel =
      EventChannel('flutter.baseflow.com/geolocator_updates');

  /// The event channel used to receive [NmeaMessage] updates from the native
  /// platform.
  @visibleForTesting
  EventChannel nmeaEventChannel = EventChannel('flutter.baseflow.com/nmea_updates');
```